app/models/user.rb

```diff
@@ -219,6 +219,19 @@ def create_home_room
     update_attributes(main_room: room)
   end
 
+  # returns true if the user has attempted to log in too many times in the past 24 hours
+  def locked_out?
+    attempts = failed_attempts.to_i
+    within_1_day = (24.hours.ago..DateTime.now).cover?(last_failed_attempt)
+
+    return true if attempts > 5 && within_1_day
```

Comment:
Please consider making this magic number configurable.

Reply:
This is a massiv attack vector for a denial of service. Please add a config option which also allows to disable this behavior.

I would prefer a throttle, with increasing time to a maximum of one hour.

Reply:
@sualko imho a proper solution, which afaik most of the bug BBB providers already have deployed, is to use for axample Nginx rate limiting in front of these critical URLs of Greenlight.